hugo/plan.sh

```diff
@@ -1,39 +1,35 @@
 pkg_name=hugo
 pkg_origin=core
-pkg_version="0.46"
+pkg_version="0.49.2"
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=("Apache-2.0")
 pkg_description="Hugo is one of the most popular open-source static site generators."
 pkg_build_deps=(
-  core/dep
-  core/mage
+  core/go
+  core/git
+  core/gcc
 )
 pkg_deps=(
   core/glibc
 )
 pkg_bin_dirs=(bin)
-pkg_source="https://github.com/gohugoio/hugo"
+pkg_repository="https://github.com/gohugoio/hugo.git"
```

Comment:
I'm curious about this change. Could you explain the intent behind changing `pkg_source` to `pkg_repository` for my edification?

Small 🚲 🏠 ,  I don't think `pkg_repository` is a variable provided by plan-build. I _think_ I'd prefer to see it prefixed with an` _` to indicate that the variable is something unique to this plan and not a Habitat construct.   Thoughts?

Reply:
Including `pkg_source` will make it want to `do_download`, using something like `pkg_repository` indicates more clearly what is being stored here, a link to the repository which is cloned later in the build step. Its just a variable, not one used by Habitat, but one named to represent the contents of that variable. Its the packages repository.

Another common example is `pkg_distname`, not necessarily one used by habitat, but common through plans, and follows the habitat approach of `pkg_` prefixing.